File: Api/Betto.Model/DTO/TeamDTO.cs
using Betto.Model.Entities;

namespace Betto.Model.DTO
{
    public class TeamDTO
    {
        public int TeamId { get; set; }
        public int LeagueId { get; set; }
        public string Name { get; set; }
        public string Logo { get; set; }
        public string Country { get; set; }
        public string IsNational { get; set; }
        public VenueDTO Venue { get; set; }

        public static explicit operator TeamDTO(TeamEntity team)
            => team == null ? null : new TeamDTO
            {
                TeamId = team.TeamId,
                LeagueId = team.LeagueId,
                Name = team.Name,
                Logo = team.Logo,
                Country = team.Country,
                IsNational = team.IsNational,
                Venue = (VenueDTO)team.Venue
            };
    }
}

File: Web/src/shared/TeamService/team-dto.js
export class TeamDTO {
    constructor(team) {
        this.teamId = team.teamId;
        this.leagueId = team.leagueId;
        this.name = team.name;
        this.logo = team.logo;
        this.country = team.country;
        this.isNational = team.isNational;
        this.venue = team.venue;
    }
}

export default {
    TeamDTO
}